src/main/scala/mesosphere/marathon/core/task/tracker/TaskTracker.scala

```diff
@@ -35,5 +35,69 @@ trait TaskTracker {
 }
 
 object TaskTracker {
-  case class App(appName: PathId, tasks: Iterable[MarathonTask])
+  /**
+    * Contains all tasks grouped by app ID.
+    */
+  case class AppDataMap private (appTasks: Map[PathId, TaskTracker.App]) {
```

Comment:
I added that to https://github.com/mesosphere/marathon/issues/3003, so we have a pure renaming PR.